.github/workflows/push-tag-create-release.yml

```
@@ -95,14 +95,14 @@ jobs:
           --output auto-changelog.md \
           --template .changeset/templates/patch-changelog.hbs
       - name: Generate changelog
-        # This changelog is a basically empty one used for everything except client.
-        if: fromJson(env.RELEASE_JSON).releaseType != 'patch' || fromJson(env.RELEASE_JSON).packageOrReleaseGroup != 'client'
+        # This changelog is a basically empty one used for everything except client patches and build-tools.
+        if: ${{ !(fromJson(env.RELEASE_JSON).releaseType == 'patch' && fromJson(env.RELEASE_JSON).packageOrReleaseGroup == 'client') && fromJson(env.RELEASE_JSON).packageOrReleaseGroup != 'build-tools' }}
         run: |
           echo "This is a **${{ fromJson(env.RELEASE_JSON).releaseType }}** release." > auto-changelog.md
 
-      # Only creates GH releases for client releases.
+      # Only creates GH releases for client and build-tools releases.
       - name: Create GH release
-        if: fromJson(env.RELEASE_JSON).packageOrReleaseGroup == 'client'
+        if: fromJson(env.RELEASE_JSON).packageOrReleaseGroup == 'client' || fromJson(env.RELEASE_JSON).packageOrReleaseGroup == 'build-tools'
```

```suggestion
        if: contains(fromJSON('["client", "build-tools"]'), fromJson(env.RELEASE_JSON).packageOrReleaseGroup)
```
I like it better, but feel free to skip

---

Leaving as-is for consistency with the other uses. I find it easier to make changes to the logic (which will have to happen as we roll this out) if I use the same string for the same logical expression. YMMV.